modules/core/src/lib/attribute-manager.js

```diff
@@ -313,7 +318,7 @@ export default class AttributeManager {
       const attribute = attributes[attributeName];
 
       // Initialize the attribute descriptor, with WebGL and metadata fields
-      newAttributes[attributeName] = new Attribute(
+      const newAttribute = (newAttributes[attributeName] = new Attribute(
```

Comment:
Nit: So far we have avoided double assignments. Assuming the parenthesis here were added to defeat the linter rule?

If you really prefer to use such syntax use `// eslint-disable-next-line ...` instead and be explicit about it.